public/app/features/scenes/apps/scenes.tsx

```diff
@@ -154,14 +154,23 @@ export function getHttpHandlerListScene(): EmbeddedScene {
         }),
         body: new SceneFlexLayout({
           direction: 'row',
+          key: `row-${frameIndex}`,
           children: [
             new SceneFlexItem({
+              key: `flex1-${frameIndex}`,
```

Comment:
Wonder why you use explicit keys?

Reply:
@dprokop  because otherwise when you cycle between different drilldown handlers the drilldown panel flicks are this layout item is unmounted (layout adjusts) then re-added